.github/workflows/test.yml

```
@@ -1,26 +1,56 @@
 name: Tests
 on: [push, pull_request]
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      - name: Install requirements
+        run: pip install flake8 pycodestyle black[python2]==21.12b0 click==8.0.4 isort
```

```suggestion
        run: pip install flake8 pycodestyle black isort
```

We don't need to pin the `black` version any more because we don't want to check `python2` codestyle :grin: